.github/workflows/stale.yml

```
@@ -0,0 +1,20 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

```suggestion
    # 1:30am every day
    - cron: '30 1 * * *'
```

i hate crontab notation 😭